Makefile

```diff
@@ -20,6 +20,13 @@ machine-controller: $(shell find cmd pkg -name '*.go') vendor
 
 docker-image: machine-controller
 	docker build -t $(IMAGE_NAME) .
+	if git describe --tags $(shell git rev-parse HEAD)|grep -v -- '-g'; then \
+		$(eval IMAGE_TAG = $(shell git describe --abbrev=0 --tags)) \
```

Comment:
You can get the tag from the circle ci environment variables: https://circleci.com/docs/2.0/env-vars/#build-details

Reply:
nevermind - i'm an idiot. Obviously those are not suitable for the Makefile :man_facepalming: